src/app/shared/components/template/components/task-progress-bar/task-progress-bar.component.ts

```diff
@@ -147,6 +158,14 @@ export class TmplTaskProgressBarComponent
     return (this.subtasksCompleted / this.subtasksTotal) * 100;
   }
 
+  getStrokeOffset(): number {
```

Comment:
Again, would be good to have a comment here (JSDoc style) to indicate that this method relates to the "wheel" variant. It should also be explicitly tagged as `public`.